docs/root/configuration/http_filters/lua_filter.rst

```diff
@@ -365,3 +377,29 @@ Get bytes from the buffer. By default Envoy will not copy all buffer bytes to Lu
 cause a buffer segment to be copied. *index* is an integer and supplies the buffer start index to
 copy. *length* is an integer and supplies the buffer length to copy. *index* + *length* must be
 less than the buffer length.
+
+.. _config_http_filters_lua_metadata_wrapper:
+
+Metadata object API
+-------------------
+
+get()
+^^^^^
+
+.. code-block:: lua
+
+  metadata:get(key)
+
+Gets a metadata. *key* is a string that supplies the filter key. Returns a table that is the
```

Comment:
s/filter key/metadata key. Also, I don't think it always returns a table? Can you clarify?